core/server/data/importer/importers/data/base.js

```diff
@@ -189,12 +189,14 @@ class Base {
 
             // CASE: you import null, fallback to owner
             if (!obj[key]) {
-                if (!userReferenceProblems[obj.id]) {
-                    userReferenceProblems[obj.id] = {obj: _.cloneDeep(obj), keys: []};
+                // Exception: If the imported post is a draft published_by will be null. Not a userReferenceProblem.
+                if (key !== 'published_by' && obj.status !== 'draft') {
```

Comment:
The fix works 👍 

I am just wondering why the whole code is not executed if the key is not published_by?
That means any published_by key is skipped.

Wouldn't it be better if you ask?

```if key is published_by and status is draft return```